### ISSUE TITLE
Obfuscates

### ISSUE THREAD

**Issue by NCN** [Aralhach/bashobfus#3]
I think a variable called ``` server ``` is obfuscated to ``` a191 ``` earlier in the script. Then later:
```
if [ -z "${serverNames}" ]; then serverNames="Test1 Test2"; fi
```
becomes

```
if [ -z "${a191Names}" ]; then serverNames="Test1 Test2"; fi
```

The first instance of ```serverNames``` became ```a191Names``` but the second instance did not.

**Comment by Aralhach** [comment]
Yes, the current version of the script only substitutes variable assignments when they occur at the beginning of the line. I think it would be best to make it a better Bash parser and actually grasp the lexical tokens.  The current script is a hacky regex substituter :D But I would like to polish it up some time, hopefully soon.  Thanks a lot for running it and for your input!
